src/udev/udev-builtin-net_id.c

```diff
@@ -641,7 +689,11 @@ static int names_pci_slot(sd_device *dev, sd_device *pci_dev, const char *prefix
                          strna(domain), bus_and_slot, strna(func), strna(port),
                          special_glyph(SPECIAL_GLYPH_ARROW_RIGHT), empty_to_na(str));
 
-        r = pci_get_hotplug_slot(pci_dev, &hotplug_slot);
+        if (naming_scheme_has(NAMING_FIRMWARE_NODE_SUN))
+                r = pci_get_slot_from_firmware_node_sun(pci_dev, &slot);
+        else
+                r = pci_get_hotplug_slot(pci_dev, &slot);
+
```

Comment:
please remove this empty line. we prefer in our codebase if the function call and it's immediate error handling stays together. hence place empty lines before the func call, or after the error handling, but please do not separate func call and error handling